Snakefile

```diff
@@ -168,16 +189,17 @@ rule gene2Phenotype:
         cacheDir = config['global']['cacheDir'],
         schema = f"{config['global']['schema']}/opentargets.json"
     output:
-        ddBucket = GS.remote(f"{config['Gene2Phenotype']['inputBucket']}/DDG2P-{timeStamp}.csv.gz"),
-        eyeBucket = GS.remote(f"{config['Gene2Phenotype']['inputBucket']}/EyeG2P-{timeStamp}.csv.gz"),
-        skinBucket = GS.remote(f"{config['Gene2Phenotype']['inputBucket']}/SkinG2P-{timeStamp}.csv.gz"),
-        cancerBucket = GS.remote(f"{config['Gene2Phenotype']['inputBucket']}/CancerG2P-{timeStamp}.csv.gz"),
-        evidenceFile= GS.remote(f"{config['Gene2Phenotype']['outputBucket']}/gene2phenotype-{timeStamp}.json.gz")
+        ddBucket = 'DDG2P.csv.gz',
+        eyeBucket = 'EyeG2P.csv.gz',
+        skinBucket = 'SkinG2P.csv.gz',
+        cancerBucket = 'CancerG2P.csv.gz',
+        evidenceFile = 'gene2phenotype.json.gz'
     log:
-        GS.remote(logFile)
+        'log/gene2Phenotype.log'
     shell:
         """
-        # Retain the inputs and save to GCS.
+        exec &> {log}
+        # Retain the inputs. They will be later uploaded to GCS.
         cp {input.ddPanel} {output.ddBucket}
```

Comment:
Unfortunately as `GS.remote` was removed from the input, `cp` command doesn't work anymore.

We have to use either `curl` or `wget` eg:
```bash
wget ftp.ebi.ac.uk/pub/databases/gene2phenotype/DD/DDG2P_1_12_2021.csv.gz -O DDG2P.csv.gz
```

Reply:
Also, when calling `modules/Gene2Phenotype.py` the input files needs to be changed:

```bash
        python modules/Gene2Phenotype.py \
          --dd_panel {output.ddBucket} \
          --eye_panel {output.eyeBucket} \
          --skin_panel {output.skinBucket} \
          --cancer_panel {output.cancerBucket} \
          --output_file {output.evidenceFile} \
          --cache_dir {params.cacheDir} \
          --local
```

Reply:
Actually I wrote a reply to this one a week ago, at the same time as the previous comment, but apparently I forgot to send it :man_facepalming: 

The problems mentioned here were also caused by the same Snakemake bug when handling the remote files. Everything works now: the rule **gene2Phenotype** copies the remove files (DD, eye, skin, cancer) to local ones; then the rule **all** uploads them to retain in Google Storage, alongside all other files listed in `ALL_FILES`